guides/common/modules/proc_enabling-the-satellite-tools-repository.adoc

```diff
@@ -17,6 +17,7 @@ endif::[]
 +
 If the *{project-client-name}* items are not visible, it may be because they are not included in the Red{nbsp}Hat Subscription Manifest obtained from the Customer Portal.
 To correct that, log in to the Customer Portal, add these repositories, download the Red{nbsp}Hat Subscription Manifest and import it into {Project}.
+For more information, see {ContentManagementDocURL}managing_red_hat_subscriptions_content-management[Managing Red Hat Subscriptions] in _{ContentManagementDocTitle}_.
```

Comment:
```suggestion
For more information, see {ContentManagementDocURL}Managing_Red_Hat_Subscriptions_content-management[Managing Red Hat Subscriptions] in _{ContentManagementDocTitle}_.
```
Watch the capitalization of the anchor.

Reply:
Right. I should have verified it thoroughly.